src/stylus/components/_datatables.styl

```diff
@@ -41,30 +41,47 @@
   justify-content: flex-end
   align-items: center
   font-size: 12px
-  
+
   .btn
     color: inherit
-    
+
     &:last-of-type
       margin-left: 18px
 
   &__pagination
     text-align: center
     margin: 0 26px 0 32px
-    
+
   &__select
     display: flex
     align-items: center
     justify-content: center
-    
+
     .input-group--select
       margin: 13px 0 13px 34px
       position: initial
-      
+
       .input-group__selections__comma
         color: rgba(#000, .54) !important
         font-size: 12px
         padding-top: 6px
-        
+
       .input-group__append-icon
-        color: rgba(#000, .54) !important
\ No newline at end of file
+        color: rgba(#000, .54) !important
+
+.datatable__progress
+  tr
+    height: auto !important
+
+  th
+    padding: 0 !important
+
+    .progress-linear
+      top: -3px
+      margin: 0 0 -3px
+      opacity: 0
```

Comment:
progress linear can be hidden and displayed by using the ```active``` [prop](https://github.com/vuetifyjs/vuetify/blob/master/src/components/progress/ProgressLinear.vue#L23). Can we use that instead of writing specific styles?